### PR TITLE
Modify rule S4423: Add Docker (APPSEC-460)

### PR DESCRIPTION
**Pull request by egon-okerman-sonarsource**
Specification: [APPSEC-460](https://sonarsource.atlassian.net/browse/APPSEC-460)
Implementation: [SONARIAC-599](https://sonarsource.atlassian.net/browse/SONARIAC-599)
PoC: https://github.com/SonarSource/appsec-poc/pull/111

[Preview](https://sonarsource.github.io/rspec/#/rspec/S4423/docker)

[APPSEC-460]: https://sonarsource.atlassian.net/browse/APPSEC-460?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SONARIAC-599]: https://sonarsource.atlassian.net/browse/SONARIAC-599?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by pierre-loup-tristant-sonarsource — CHANGES_REQUESTED** [review]
I think the messages could be more precise, especially when an option is omitted.

**Comment by pierre-loup-tristant-sonarsource** [comment]
The current message we have for most rules is: "Change this code to use a stronger protocol."
In general I think it's not great and should be updated.

For the docker rule, I suggest one of the following messages:

- "Change this code to enforce TLS 1.2 or above."
- "Change this code to disable support of older TLS versions."

File: rules/S4423/docker/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "securityStandards": {
+        "CWE": [
+            327,
+            326,
+            295
+        ],
+        "OWASP": [],
+        "OWASP Mobile": [],
+        "MASVS": [],
```
[the remark this answers is not in the document]
Comment (egon-okerman-sonarsource) [reply rules/S4423/docker/metadata.json:10]: Added to our backlog: [APPSEC-488](https://sonarsource.atlassian.net/browse/APPSEC-488)

**Review by pierre-loup-tristant-sonarsource — APPROVED** [review]
LGTM

**Comment by pierre-loup-tristant-sonarsource** [comment]
@egon, you can maybe rebase the PR to make sure the `validate_links` check is passing.

**Pull request merged**